src/brain/issueLabelHandler/followups.ts

```diff
@@ -158,5 +169,25 @@ export async function ensureOneWaitingForLabel({
 
   await modifyProjectIssueField(itemId, labelName, STATUS_FIELD_ID, octokit);
 
+  let timeToRespondBy;
+  if (labelName === WAITING_FOR_PRODUCT_OWNER_LABEL) {
+    timeToRespondBy =
+      (await calculateSLOViolationTriage(UNTRIAGED_LABEL, issue.labels)) ||
+      moment().toISOString();
+  } else if (labelName === WAITING_FOR_SUPPORT_LABEL) {
+    timeToRespondBy =
+      (await calculateSLOViolationRoute(UNROUTED_LABEL)) ||
+      moment().toISOString();
+  } else {
+    timeToRespondBy = '';
+  }
```

Comment:
Probably not the biggest deal, but aren't we running `calculateSLOViolation*` twice, once for the comment and once for the project field? Won't they be slightly different values when conceptually they should be the same?

Reply:
Yes, for now they may differ by a factor of milliseconds but for our purposes that shouldn't be a big deal since to our logic and we will remove the timestamp from comments soon. I don't see a way where we can fold that logic into one right now since they exist in two different webhook listeners